### ISSUE TITLE
Confusions about the identification between DIA-NN(1.7.12) and DIA-NN(1.8)

### ISSUE THREAD

**Issue by Clovernana** [vdemichev/DiaNN#358]
Hello, 

Thank you for your great work!
 I have been using the software to analyze the DIA data. But I'm confused about the identifiactions of the two different wersions (DIA-NN(1.7.12) vs DIA-NN(1.8) ).  I use the FASTA based searching strategy and the GPF-based spectral library to search my data. In the version of 1.7.12, the GPF-based or the FASTA-based strategy identified more proteins than v1.8 (Maybe due to the stringent control of global precursor and protein FDR. But I'm not clear).  The second question is that the combined serching strategy (searching different groups of data together ) identified less proteins than the identifications got by searching the different groups of data seperately. I don't know the reason for this. Could you please comment on these two questions?

Jina

**Comment by Clovernana** [comment]
OK, thank you very much!


The files(.rar) are in the attachment.


For the version of DIA-NN(v1.7.12), I didn't search them together and I didn't have a installation package for that version. But through the comparison of the three ways(v1.7.12_seprate, v1.8_combined and v1.8_separate ), it seems that v1.8 identifies more than v1.7.12 and the combined searching strategy identifies less than the separate way. But I don't know clearly why. 


Waiting for your reply!


Jina






On 4/11/2022 23:00，Vadim ***@***.***> wrote：

Yes, stringent control. It's overly stringent in 1.8 for extra reliability.
For two strategies, please share the log files, I will take a look.

Vadim

**Comment by vdemichev** [comment]
Sorry, cannot see the attachment... Can you please attach two .txt log files separately, not in .rar?

**Comment by vdemichev** [comment]
Yes, somehow MBR actually decreases the ID rate for the low injection amount samples. Very strange, I would suggest to set Library generation to "IDs, RTs and IM profiling" - looks like for whatever reason using spectra from those runs is not a good idea.

**Comment by Clovernana** [comment]
> Yes, somehow MBR actually decreases the ID rate for the low injection amount samples. Very strange, I would suggest to set Library generation to "IDs, RTs and IM profiling" - looks like for whatever reason using spectra from those runs is not a good idea.

OK, I'll try it.
